projects/plugins/jetpack/tests/e2e/specs/post-connection/free-blocks.test.js

```diff
@@ -1,45 +1,39 @@
+import { test, expect } from '../../fixtures/base-test.js';
 import {
 	BlockEditorPage,
 	PinterestBlock,
 	EventbriteBlock,
-} from 'jetpack-e2e-commons/pages/wp-admin';
-import { PostFrontendPage } from 'jetpack-e2e-commons/pages';
+} from 'jetpack-e2e-commons/pages/wp-admin/index.js';
+import { PostFrontendPage } from 'jetpack-e2e-commons/pages/index.js';
 import config from 'config';
-import { testStep } from 'jetpack-e2e-commons/reporters';
-import { prerequisitesBuilder, Plans } from 'jetpack-e2e-commons/env';
+import { prerequisitesBuilder, Plans } from 'jetpack-e2e-commons/env/index.js';
+import playwrightConfig from '../../playwright.config.cjs';
 
-/**
- *
- * @group post-connection
- * @group free-blocks
- * @group blocks
- * @group gutenberg
- * @group atomic
- */
-
-describe( 'Free blocks', () => {
+test.describe.parallel( 'Free blocks', () => {
 	let blockEditor;
 
-	beforeAll( async () => {
-		await prerequisitesBuilder()
+	test.beforeAll( async ( { browser } ) => {
+		const page = await browser.newPage( playwrightConfig.use );
```

Comment:
I noticed that connection spec does not instantiate `page` Is there a reason for that?

Reply:
Yes, `page` is not available in `beforeAll` and `afterAll`, because Playwright ensures context isolation for each test, so `page` is only available in `beforeEach` hooks and `test`. This clashes a bit with our current approach, but I actually find it cleaner this way. In a follow-up PR, I'm thinking of trying to replace our `beforeAll` hooks with fixtures.